mackup/applications/illustrator.cfg

```diff
@@ -11,3 +11,20 @@ Library/Preferences/Adobe Illustrator 18 Settings
 Library/Preferences/Adobe Illustrator 19 Settings
 Library/Preferences/Adobe Illustrator 20 Settings
 Library/Preferences/com.adobe.illustrator.plist
+
+Library/Preferences/Adobe Illustrator 23 Settings/en_US/Adobe Illustrator Prefs
```

Comment:
If you look up, the whole folders have been synced. Why not doing the same for 23?

Reply:
Hm! I agree. I wasn't sure if those whole folders *would* get synced. But absolutely, top level folders is what we want.